schema/models.py

```diff
@@ -161,19 +148,23 @@ class Meta:
 
 class MemberGroup(models.Model):
 	id = models.AutoField(primary_key=True)
-	member = models.ForeignKey(settings.AUTH_USER_MODEL)
-	group = models.ForeignKey('Group')
+	member = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+	group = models.ForeignKey('Group', on_delete=models.CASCADE)
 	timestamp = models.DateTimeField(auto_now=True)
 	admin = models.BooleanField(default=False)
-	moderator = models.BooleanField(default=False)
+	all_emails = models.BooleanField(default=True)
+	digest = models.BooleanField(default=False)
 	no_emails = models.BooleanField(default=False)
-	always_follow_thread = models.BooleanField(default=True)
```

Comment:
As I merged this code from the previous PR, I got this message as I migrate to the newer DB: 
```
Did you rename membergroup.always_follow_thread to membergroup.admin_emails (a BooleanField)? [y/N] 
Did you rename membergroup.always_follow_thread to membergroup.all_emails (a BooleanField)? [y/N]
```

Can you let me know which one is y and N?

Reply:
always_follow_thread was changed to all_emails so it would be N to admin_emails and y to all_emails.